pom.xml

```diff
@@ -133,6 +134,11 @@ file, You can obtain one at http://mozilla.org/MPL/2.0/.
             <artifactId>guice</artifactId>
             <version>${dependency.guice.version}</version>
         </dependency>
+        <dependency>
```

Comment:
What is this new dependency used for?

Reply:
This is used to add the assisted parameter ```testType``` to the```CustomPrometheusCollector``` class. This way we can invoke the right instance of the class using guice's ```FactoryModuleBuilder```